### ISSUE TITLE
Scenario indentation

### ISSUE THREAD

**Issue by PetrHavelka** [jiridudekusy/uuDockitHelper#3]
Please fix problem with scenario indentation. In case of scenario has more than 9 steps indentation has to be done with 2 tabs and 1 space. Editing of long existing scenario will crash its indentation.

For example:
```
1.  Step 1
2.  Step 2
...
9.  Step 9
10.  Step 10
    1.  Step 10.1 - incorrect - it will be step 11.
11.  Step 11
     1.  Step 11.1 - correct - it has one more space at the begining of the row
```

**Comment by jiridudekusy** [comment]
Now up to 99 items in each ordered list are supported.

**Comment by jiridudekusy** [comment]
Fix #3
